plugins/javadoc/src/test/kotlin/org/jetbrains/dokka/javadoc/JavadocIndexTest.kt

```diff
@@ -21,7 +21,7 @@ internal class JavadocIndexTest : AbstractJavadocTemplateMapTest() {
             AnnotationTarget.ANNOTATION_CLASS::class.java.methods.any { it.name == "describeConstable" }
 
         testIndexPages { indexPages ->
-            assertEquals(if (hasAdditionalFunction()) 41 else 40, indexPages.sumBy { it.elements.size })
+            assertEquals(if (hasAdditionalFunction()) 32 else 31, indexPages.sumBy { it.elements.size })
```

Comment:
Out of curiosity, why such a drop in numbers? Where did 10 functions go? :))

Reply:
These are synthetic methods (toString, etc.) and they got merged, earlier, the `inheritedFrom` DRI was differing because it was not always pointing to Object/Any